docs/rules/no-disallowed-lwc-imports.md

```diff
@@ -39,3 +39,21 @@ import { LightningElement, wire, api } from 'lwc';
 ```
 
 If you disable this rule, then you may import unstable or otherwise undesirable APIs from `lwc`.
+
+### `allowList`
```

Comment:
I would recommend changing this to `allowlist` (lowercase "l"). It's definitely a neologism (I can't find it in any English dictionary), but when I google it, the most common spelling is `allowlist`, not `allowList`.